test/dlc_tests/container_tests/bin/testTensorFlow

```diff
@@ -3,7 +3,7 @@
 HOME_DIR=/test
 BIN_DIR=${HOME_DIR}/bin
 LOG_DIR=${HOME_DIR}/logs
-PROFILER_LOG_DIR="/logs/train/plugins/profile"
+PROFILER_LOG_DIR="/logs/plugins/profile"
```

Comment:
I ran the test locally and there is no plugins dir in "train" - we need to get to the bottom of why, though, before accepting this change

Reply:
Additionally, if this is in fact behavior change from TF2.6 to TF2.7, we need to handle this in a way that's backward compatible

Reply:
@hsl89 why is this resolved? Can you add in a check to conditionally set this path based on TF version?